site/content/en/docs/api_sdk/sdk/lowlevel-api.md

```diff
@@ -0,0 +1,388 @@
+    # Parameters can be passed as a plain dict with JSON-serialized data
+    # or as model objects (from cvat_sdk.api_client.models), including
+    # mixed variants.
+    #
+    # In case of dicts, keys must be the same as members of models.I<ModelName>
+    # interfaces and values must be convertible to the corresponding member
+    # value types (e.g. a date or string enum value can be parsed from a string).
+    #
+    # In case of model objects, data must be of the corresponding
+    # models.<ModelName> types.
+    #
+    # Let's use a dict here. It should look like models.ITaskWriteRequest
+    task_spec = {
+        'name': 'example task',
+        "labels": [{
+            "name": "car",
+            "color": "#ff00ff",
+            "attributes": [
+                {
+                    "name": "a",
+                    "mutable": True,
+                    "input_type": "number",
+                    "default_value": "5",
+                    "values": ["4", "5", "6"]
+                }
+            ]
+        }],
+    }
+
+    try:
+        # Apis can be accessed as ApiClient class members
+        # We use different models for input and output data. For input data,
+        # models are typically called like "*Request". Output data models have
+        # no suffix.
+        (task, response) = api_client.tasks_api.create(task_spec)
+    except exceptions.ApiException as e:
+        # We can catch the basic exception type, or a derived type
+        print("Exception when trying to create a task: %s\n" % e)
+
+    # Here we will use models instead of a dict
+    task_data = models.DataRequest(
+        image_quality=75,
+        start_frame=2,
+        stop_frame=5,
+        client_files=[
+            open('image1.jpg', 'rb'),
+            open('image2.jpg', 'rb'),
+        ],
+    )
+
+    # If we pass binary file objects, we need to specify content type.
+    # For this endpoint, we don't have response data
+    (_, response) = api_client.tasks_api.create_data(task.id,
+        data_request=task_data,
+        _content_type="multipart/form-data",
+
+        # we can choose to check the response status manually
+        # and disable the response data parsing
+        _check_status=False, _parse_response=False
+    )
+    assert response.status == 202, response.msg
+
+    # Wait till task data is processed
+    for _ in range(100):
+        (status, _) = api_client.tasks_api.retrieve_status(task.id)
+        if status.state.value in ['Finished', 'Failed']:
+            break
+        sleep(0.1)
+    assert status.state.value == 'Finished', status.message
+
+    # Update the task object and check the task size
+    (task, _) = api_client.tasks_api.retrieve(task.id)
+    assert task.size == 4
+```
+
+## Available API Endpoints
+
+All URIs are relative to _`http://localhost`_
+
+APIs can be instanted directly like this:
+
+```python
+from cvat_sdk.api_client import ApiClient, apis
+
+api_client = ApiClient(...)
```

Comment:
I don't think that we should describe all possible ways to use cvat_sdk.api_client. We need to show in the documentation only the recommended way.